From cb6ddee486261bc8e040cc770876a993c97ff38f Mon Sep 17 00:00:00 2001
From: Haoran Wang <b50027@freescale.com>
Date: Mon, 4 May 2015 15:21:09 +0800
Subject: [PATCH 1/2] MA-6611 Remove no used drawable quote

The icon "ic_next" is not available in drawable.
Remove it for the normal layout. Or it will cause
the FileNotFoundException when startup the apk.

Signed-off-by: Haoran Wang <b50027@freescale.com>
---
 res/layout/account_setup_template.xml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/res/layout/account_setup_template.xml b/res/layout/account_setup_template.xml
index affe9c9..18b0b24 100644
--- a/res/layout/account_setup_template.xml
+++ b/res/layout/account_setup_template.xml
@@ -82,8 +82,6 @@
             android:layout_height="wrap_content"
             android:layout_width="wrap_content"
             android:layout_gravity="center_vertical"
-            android:drawableEnd="@drawable/ic_next"
-            android:drawableRight="@drawable/ic_next"
             android:drawablePadding="@dimen/setup_nav_button_drawable_padding"
             android:text="@string/next"
             style="@style/account_setup_nav_button" />
-- 
1.8.0

